turbopy/core.py

```diff
@@ -363,10 +382,29 @@ def parse_grid_data(self):
             self.set_value_from_keys("dr", {"dr", "dx"})
             self.num_points = 1 + (self.r_max - self.r_min) / self.dr
             if not (self.num_points % 1 == 0):
-                raise (RuntimeError("Invalid grid spacing: configuration does not imply integer number of grid points"))
+                raise (RuntimeError("Invalid grid spacing: configuration "
```

Comment:
Wow! I had no idea you could join strings this way! TIL new syntax for defining long strings.